tests/python/pants_test/init/test_util.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+import os
+from contextlib import contextmanager
+
+from pants.fs.fs import safe_filename_from_path
+from pants.init.util import init_workdir
+from pants.option.option_value_container import OptionValueContainer
+from pants.testutil.test_base import TestBase
+from pants.util.contextutil import temporary_dir
+
+
+class UtilTest(TestBase):
+    @contextmanager
+    def physical_workdir_base(self) -> OptionValueContainer:
+        with temporary_dir(cleanup=False) as physical_workdir_base:
+            bootstrap_options = self.get_bootstrap_options(
+                [f"--pants-physical-workdir-base={physical_workdir_base}"]
+            )
+            yield bootstrap_options
+
+    def assertExists(self, path):
```

Comment:
This was my bad, but I think this should actually be snake case.
```suggestion
    def assert_exists(self, path):
```